addons/mail/static/src/components/messaging_menu/messaging_menu.scss

```diff
@@ -29,7 +29,8 @@
         flex: 0 1 auto;
         width: 350px;
         min-height: 50px;
-        max-height: 400px;
+        // Sass is case-sensitive, but CSS isn’t. You can not use cals in min() saas function
```

Comment:
Took me a while to understand the trick. I now understand the comment, but it could be better explained so that everyone understand what's going on here.

Suggestion:

```scss
min-height: 50px;
/**
 * Note: Min() refers to CSS min() and not SCSS min().
 *
 * We want CSS min() and not SCSS min() because the former supports calc while the latter doesn't.
 * To by-pass SCSS min() shadowing CSS min(), we rely on SCSS being case-sensitive while CSS isn't.
 * As a result, Min() is ignored by SCSS while CSS interprets as its min() function.
 */
max-height: Min(calc(100vh - 140px), 630px);
```

Reply:
@alexkuhn .. Changes Done.
Thanks for the suggestions.